src/auth/auth-token.entity.ts

```diff
@@ -21,7 +21,9 @@ export class AuthToken {
   @Column({ unique: true })
   keyId: string;
 
-  @ManyToOne((_) => User, (user) => user.authTokens)
+  @ManyToOne((_) => User, (user) => user.authTokens, {
+    onDelete: 'CASCADE',
```

Comment:
Please also document the usages of 'cascade'

Reply:
I did so